design/specs/table-import.md

```diff
@@ -1,166 +1,107 @@
-
-For example, if calling the add new table function from the sidebar menu, the context will determine that the system must add the table to that particular schema. 
-
-However, a user might choose to add a new table from a different context, where there's no pre-determined option. In this case, a user would have to select the desired option. 
-
-Making pre-determined options visible and editable in the interface can be helpful for users when confirming outcomes where contextual cues are missing.
-
-# User Interface (WIP)
-## Components
-### Input Controls
-Input controls allow the user to enter or select different values needed to perform tasks within Mathesar. In file import, they will enable the user to specify column types and parameters required to map columns.
-
-#### Text input
-![](/assets/design/specs/table-import/rJoUmDX5_.png =140x)
-
-#### Select input
-![](/assets/design/specs/table-import/rkavQwXc_.png =140x)
-
-#### Lookup input
-![](/assets/design/specs/table-import/BysumDQqu.png =140x)
-
-#### Radio buttons
-![](/assets/design/specs/table-import/H1P5XPQcu.png =320x)
-
-#### Checkbox input
-![](/assets/design/specs/table-import/HJnhQP7cd.png =200x)
-
-#### States
-- Default
-- On Focus
-- Disabled
-
-### Buttons
-Buttons allow users to perform actions and obtain a result from the system.
-
-#### Types
-There are different types of buttons, and their use depends on several factors, e.g., the type of task they are to perform and their context.
-
-![](/assets/design/specs/table-import/rkEwqImqd.png =200x)
-
-Primary buttons are used for actionable (e.g., submit content, apply a change) and some exceptional navigational purposes (e.g., continue to a next step). However, their use should be restricted to priority or practical actions, with the ideal number of primary buttons being one per view.
-
-Secondary buttons are used alone or in combination with primary controls to complement priority actions.
-
-
-### Dialogs
-Dialogs are a type of modal window that appears in front of content and is used when the system requires users to make decisions.
-When active, dialogs will prevent other content from being interacted with. In most cases, they are driven by user action, and their unprompted use is discouraged.
-
-![](/assets/design/specs/table-import/Hk02qLQc_.png =280x)
-
-### System Notifications
-
-#### Error Messages
-![](/assets/design/specs/table-import/SJBTzvX9d.png)
+If the table is unconfirmed, there will be a warning icon on the side of table listing which when hovered on will have a confirm warning. When clicked on the table listing the user will be taken to confirmation stage.
 
+### Usage of an unconfirmed table
+![](https://share.balsamiq.com/c/kLkhvgR9Uk1njGbmgh9msB.png)
 
+If the user tries to use an unconfirmed table, there will be a modal for confirming the table where the user can directly confirm the table or go to the confirmation view. They won't be able to use it without confirming it.
```

Comment:
It is not clear to me what 'View Table' will do. Will this force confirmation? If so, can we update the action to reflect that?